pkg/log/teelogger/teelogger.go

```diff
@@ -0,0 +1,31 @@
+// Package filetee provides a go-kit compatible log mirroring tool.
+package teelogger
+
+import (
+	"github.com/go-kit/kit/log"
+)
+
+type teeLogger struct {
+	loggers []log.Logger
+}
+
+func New(loggers ...log.Logger) log.Logger {
+	l := &teeLogger{
+		loggers: loggers,
+	}
+
+	return l
+}
+
+// Log will log to each logger. If any of them error, it will return a
+// random error.
```

Comment:
doesn't this actually return the _last_ error returned by a logger, rather than a random error? iiuc the ordering would depend on the order of the loggers inserted...

Reply:
I mean... Yes, but I don't want to make any real promises here. The mirroring seems reasonable, but the error handling is janky and subject to change.